pandas/core/frame.py

```diff
@@ -8604,12 +8604,15 @@ def pivot(
             hierarchical columns whose top level are the function names
             (inferred from the function objects themselves)
             If dict is passed, the key is column to aggregate and value
-            is function or list of functions.
+            is function or list of functions. If margin = ``True``,
```

Comment:
```suggestion
            is function or list of functions. If ``margin=True``,
```

Just to be consistent with the one below.